packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -257,16 +249,30 @@ export default function MainFeedLayout({
     </LayoutHeader>
   );
 
+  const hasFiltered = feedName === MainFeedPage.MyFeed && !alerts?.filter;
+
   const header = (
     <LayoutHeader className="flex-col">
-      {shouldShowMyFeed && alerts?.filter && (
+      {alerts?.filter && (
         <CreateMyFeedButton
           action={() => setCreateMyFeed(true)}
           flags={flags}
         />
       )}
       <div className="flex flex-row flex-wrap gap-4 items-center mr-px w-full h-[3.125rem]">
-        <h3 className="flex flex-1 typo-headline">{feedTitles[feedName]}</h3>
+        <h3 className="flex flex-row flex-1 items-center typo-headline">
+          {feedTitles[feedName]}
+          {hasFiltered && (
+            <SimpleTooltip placement="right" content="Feed filters">
+              <Button
+                iconOnly
+                className="mx-3 btn-tertiary"
+                icon={<FilterIcon />}
+                onClick={() => setIsFeedFiltersOpen(true)}
+              />
+            </SimpleTooltip>
```

Comment:
Perhaps we can extract this into `FeedFilterButton` (even in this file) for readability

Reply:
Totally agree with that. Also, I will have to migrate the alert for my feed when the user has activated the feature. I will probably include that part in this new component.

Reply:
Refactored this in the PR related to the pointer 🚢